.changeset/six-coins-matter.md

```diff
@@ -0,0 +1,4 @@
+---
+'@finos/legend-graph': minor
+'@finos/legend-studio': minor
+---
```

Comment:
Please split this changeset into 2.
1. @finos/legend-studio: patch
2. @finos/legend-graph: minor (with the message about what you added support for)